bzira.py

```diff
@@ -72,13 +87,17 @@ def bz_to_jira_version(options, bug):
 
     if bug.product in bzprod_version_map:
         b2j = bzprod_version_map[bug.product]
-        
-        print bug.product + "/" + bzversion + "->" + b2j(bzversion)
+        if (b2j(bzversion) == "UNKNOWN"):
+            print "[WARN] Unknown mapper for " + bug.product + " / " + bzversion + " -> " + b2j(bzversion)
+        elif (bzversion == "---"):
+            print "[WARN] Target milestone not set for " + bug.product + " / " + bzversion
```

Comment:
this isn't a warning is it ? it is expected and should just be blank, right ? put behind (options.verbose) maybe ?